packages/js-moi-logic/__tests__/utils/utils.ts

```diff
@@ -1,11 +1,15 @@
+import { BaseProvider, JsonRpcProvider, VoyageProvider } from "js-moi-providers";
 import { Signer } from "js-moi-signer";
 import { Wallet } from "js-moi-wallet";
-import { JsonRpcProvider } from "js-moi-providers";
 
-export const initializeWallet = async (provider: JsonRpcProvider, mnemonic: string): Promise<Signer> => {
-    const derivationPath = "m/44'/6174'/0'/0/1";
+export const initializeWallet = async (provider: BaseProvider, mnemonic: string): Promise<Signer> => {
```

Comment:
Use `JsonRPCProvider` for testing.

Reply:
1f3b5e6479daa109ff17ab3bc7100926914b4747